pkg/registry/registry_test.go

```diff
@@ -153,6 +153,15 @@ func TestListGithubArch(t *testing.T) {
 	assert.Contains(t, platforms, Platform{Architecture: "arm64", OS: "linux"})
 }
 
+func TestListGithubNoeArch(t *testing.T) {
+	// This image requires the application/vnd.oci.image.manifest.v1+json Accept header to successfully return all architectures
+	platforms, err := DefaultRegistry.ListArchs(context.Background(), "", "ghcr.io/adevinta/noe:latest")
```

Comment:
We have seen that for some reason alpine images tends to fail more frequently maybe a good idea to do a test with alpine

Reply:
The problem tested here (missing Accept headers) is reproducible 100% of the time.

The problem does not appear with alpine on Docker Hub

testing noe image without Accept headers
<img width="1633" alt="Screenshot 2024-03-08 at 20 20 59" src="https://github.com/adevinta/noe/assets/6061440/433a5214-9a08-4b61-9249-e192b040242e">

testing alpine image without Accept headers
<img width="1636" alt="Screenshot 2024-03-08 at 20 20 42" src="https://github.com/adevinta/noe/assets/6061440/8cd9437c-3d2d-42a9-a174-cbbdc724e8b6">


testing noe image with Accept headers
<img width="1631" alt="Screenshot 2024-03-08 at 20 20 11" src="https://github.com/adevinta/noe/assets/6061440/3205b3b3-a8e6-4e9b-9ce5-e123c85decf0">

testing alpine image with Accept headers

<img width="1636" alt="Screenshot 2024-03-08 at 20 23 01" src="https://github.com/adevinta/noe/assets/6061440/78bcc928-1a79-484b-b542-8f8f86825dc5">

Additionally, the alpine image is already [tested](https://github.com/adevinta/noe/blob/d6f9a1bede8ebbc6f96a1f60d53177ca359f4117/pkg/registry/registry_test.go#L128) as a [well-known multi-arch image](https://github.com/adevinta/noe/blob/d6f9a1bede8ebbc6f96a1f60d53177ca359f4117/pkg/registry/registry_test.go#L16C7-L16C30)